training/amd/Containerfile

```diff
@@ -0,0 +1,8 @@
+ARG BASEIMAGE="quay.io/centos-bootc/centos-bootc:stream9"
+
+FROM ${BASEIMAGE}
```

Comment:
I would prefer to use `FROM quay.io/centos-bootc/centos-bootc:stream9` and use `--from $(FROM) in the Makefile like we do in the rest of the repo.

Reply:
Done